NuKeeper/Engine/RepositoryUpdater.cs

```diff
@@ -119,11 +119,11 @@ private void GitCloneToTempDir()
         {
             foreach (var current in updateSet.CurrentPackages)
             {
-                var updater = current.Path.PackageReferenceType == PackageReferenceType.ProjectFile
-                    ? (INuGetUpdater) new NuGetUpdater()
-                    : new PackagesConfigUpdater();
+                var updateCommand = current.Path.PackageReferenceType == PackageReferenceType.ProjectFile
+                    ? (IUpdatePackageCommand) new DotNetUpdatePackageCommand()
```

Comment:
The hard cast is needed to get the correct type out of the `? :` operator. But I really don't mind which way, I can change it to remove the `? :`.